src/ProjectSystemTools/BuildLogging/Model/Build.cs

```diff
@@ -54,12 +55,17 @@ public void Finish(bool succeeded, DateTime time)
             Elapsed = time - StartTime;
         }
 
-        public void Close(string logPath)
+        public void PreserveLogfile(string logPath)
```

Comment:
>PreserveLogfile [](start = 20, length = 15)

Since copying the file is no longer contained within this class (since the Roslyn log doesn't use it), we should push this copy up into the callers and remove this method. Then there will just be SetLogPath.